samples/language/Array.qs

```diff
@@ -23,16 +23,26 @@ namespace MyQuantumApp {
 
         // A basic Int Array literal
         let intArray : Int[] = [1, 2, 3, 4];
+        Message($"{intArray}");
+        Message($"{intArray} of length {Length(intArray)}");
```

Comment:
```suggestion
        Message($"Integer Array : {intArray} of length {Length(intArray)}");
```

Nice addition of showing length :-)

[Minor suggestion] We can get rid of the first message. I would recommending adding some text to make it slightly more user friendly :-)

Reply:
+1 on printing this array only once.

I wouldn't go all out to add text to every message. The goal of adding printing is to be able to see the values easily and to confirm that they match the comments, but the comments take the brunt of communication about types and language nuances here, so we don't need to duplicate that.